src/workloads/query/TimeseriesTSBSOptimizations.yml

```diff
@@ -0,0 +1,286 @@
+  The data is preloaded by the dsi configuration. See 'configurations/test_control/test_control.tsbs_query.yml' for all 
+  the details. Below are helpful statistics about the collection:
+  1. There are 20736000 documents and 161448 buckets in the collection. 
+  2. "tags" is the metaField and "time" is the timeField.
+  2. The 50th percentile of the number of documents per bucket is 93.
+  4. The 80th percentile of the number of document per bucket is 274.96.
+  3. 40435 buckets are version 1 and 121013 buckets are version 2.
+  4. Predicates of bucket at a time window of 1 hour is expected to hit 0.2% of buckets.
+  5. Predicates on a single measurement value, such as {usage_nice: 5} will hit around 18% of buckets.
+  6. The earliest date is  "2016-01-01T00:00:00Z" and the latest date is "2016-01-24T23:59:50Z".
+
+Keywords:
+  - timeseries
+  - aggregate
+  - group
+  - sort
+
+GlobalDefaults:
+  Database: &database cpu
+  Collection: &collection point_data
+  maxPhases: &maxPhases 10
+  repeat: &repeat 50
+
+Actors:
+  # The data is already loaded into the collection from the dsi configuration.
+  - Name: Quiesce
+    Type: QuiesceActor
+    Threads: 1
+    Database: *database
+    Phases:
+      OnlyActiveInPhases:
+        Active: [0]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+
+# The following two tasks test the $group rewrite for the $min and $max accumulators.
+  - Name: GroupRewriteMax
+    Type: CrudActor
+    Database: *database
+    Threads: 10
+    Phases:
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                Pipeline: [
+                  { 
+                    $group: 
+                    { 
+                      _id: "$tags",  
+                      max_usage_user: { $max: "$usage_user" },
+                      max_usage_iowait: { $max: "$usage_iowait" },
+                      max_usage_irq: { $max: "$usage_irq" },
+                      max_usage_steal: {$max: "$usage_steal"}
+                    }
+                  }
+                ]
+
+  - Name: GroupRewriteMin
+    Type: CrudActor
+    Database: *database
+    Threads: 10
+    Phases:
+      OnlyActiveInPhases:
+        Active: [2]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                Pipeline: [
+                  { 
+                    $group: 
+                    { 
+                      _id: "$tags.hostname",  
```

Comment:
Is it intentional that `GroupRewriteMax` groups by `$tags` and `GropuRewriteMin` groups by `$tags.hostname`?